GLAA.Web/Controllers/EligibilityController.cs

```diff
@@ -1,5 +1,5 @@
-﻿using GLAA.Services;
```

Comment:
If we rename the model we should rename this controller too

Reply:
I agree. Thinking about it there shouldn't be too many problems with routing so I'm happy to do it.